File: app/src/main/java/com/example/krystianwsul/organizator/notifications/TimeChangeReceiver.java
package com.example.krystianwsul.organizator.notifications;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class TimeChangeReceiver extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {
        TickService.startService(context);
    }
}

File: app/src/main/java/com/krystianwsul/checkme/ticks/TimeChangeReceiver.kt
package com.krystianwsul.checkme.ticks

import android.annotation.SuppressLint
import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class TimeChangeReceiver : BroadcastReceiver() {

    @SuppressLint("UnsafeProtectedBroadcastReceiver")
    override fun onReceive(context: Context, intent: Intent) {
        Ticker.tick("TimeChangeReceiver", true).subscribe()
    }
}
